plugins/statistics.koplugin/readerprogress.lua

```diff
@@ -391,63 +392,38 @@ function ReaderProgress:genSummaryWeek(width)
         align = "center",
         CenterContainer:new{
             dimen = Geom:new{ w = tile_width, h = tile_height },
-            TextWidget:new{
-                padding = Size.padding.default,
-                text = _("Total"),
+            TextBoxWidget:new{
+                alignment = "center",
+                text = _("Total\nPages"),
                 face = self.small_font_face,
+                width = tile_width * 0.95,
             },
         },
         CenterContainer:new{
             dimen = Geom:new{ w = tile_width, h = tile_height },
-            TextWidget:new{
-                text = _("Total"),
+            TextBoxWidget:new{
+                alignment = "center",
+                text = _("Total\nTime"),
                 face = self.small_font_face,
+                width = tile_width * 0.95,
             },
         },
         CenterContainer:new{
             dimen = Geom:new{ w = tile_width, h = tile_height },
-            TextWidget:new{
-                text = _("Average"),
+            TextBoxWidget:new{
+                alignment = "center",
+                text = _("Average\nPages"),
                 face = self.small_font_face,
+                width = tile_width * 0.95,
             }
         },
         CenterContainer:new{
             dimen = Geom:new{ w = tile_width, h = tile_height },
-            TextWidget:new{
-                text = _("Average"),
-                face = self.small_font_face,
-            }
-        }
-    }
-
-    local titles_group = HorizontalGroup:new{
-        align = "center",
-        CenterContainer:new{
-            dimen = Geom:new{ w = tile_width, h = tile_height },
-            TextWidget:new{
-                text = _("Pages"),
-                face = self.small_font_face,
-            },
-        },
-        CenterContainer:new{
-            dimen = Geom:new{ w = tile_width, h = tile_height },
-            TextWidget:new{
-                text = _("Time"),
-                face = self.small_font_face,
-            },
-        },
-        CenterContainer:new{
-            dimen = Geom:new{ w = tile_width, h = tile_height },
-            TextWidget:new{
-                text = _("Pages"),
-                face = self.small_font_face,
-            }
-        },
-        CenterContainer:new{
-            dimen = Geom:new{ w = tile_width, h = tile_height },
-            TextWidget:new{
-                text = _("Time"),
+            TextBoxWidget:new{
+                alignment = "center",
+                text = _("Average\nTime"),
```

Comment:
```suggestion
                text = _("Average\ntime"),
```

Reply:
Yes, to but is that true here, too? 
Originally these two strings (e.g. "Average" and "Time") have been in two separate `textwidgets` which have been in a vertical box. Both were capitalized.

I have just combined the two in on `textboxwidget`, so that they form the right context together. It seems they should mark a headline (and therefore they are capitalized). 

Would it be ok to let the capitalization or shall I change the second ones to lowercase?

Reply:
(Just a note that in the bottom menu, we have `Word Spacing`, `Font Size`...)

Reply:
> (and therefore they are capitalized)

Therefore nothing. :-)

>  (Just a note that in the bottom menu, we have Word Spacing, Font Size...)

Technically inconsistent, but consistent within its own little separate universe.

Reply:
But do they deserve to have their own little separate universe ?
(And did the translators really kept the capitalization on every word ?)

Reply:
I have also 2ct to add:
https://www.thoughtco.com/title-case-capitalization-1692469#:~:text=Title%20case%20is%20one%20of,up%20style%20and%20headline%20style.
and 
https://en.m.wikipedia.org/wiki/Title_case

Reply:
> (And did the translators really kept the capitalization on every word ?)

That's their choice. I should hope not, if it'd look weird in that language! ;-)

--

@zwim Yes, it *exists* and it's not uncommon. It's entirely a matter of the style guide. In our case, capitalizations are deviant.